### PR TITLE
Added interactive examples of PIX Augmentations with Google Colab

### PR DESCRIPTION
**Pull request by SupreethRao99**
Added interactive examples of running every augmentation in the pix library on a Google Colab notebook which can be found [here](https://colab.research.google.com/drive/1EJG6VNhH5aJNy9sT7jUQkgYP5GeGPUvw?usp=sharing).

### CODE REVIEW

**Comment by claudiofantacci** [comment]
Hey @SupreethRao99, thanks for the PR.
It LGTM, however I would prefer to have the `ipynb` directly under `/example` folder as `image_augmentation.ipynb`.
Then you can update the link to point to that file 😄

File: examples/README.md [hunk 1]
```diff
@@ -13,3 +13,4 @@ Then, to run the examples, you can simply run the following commands:
 $ cd examples/
 $ python image_augmentation.py
 ```
+You can also view an interactive version of the examples on [google colab](https://colab.research.google.com/drive/1EJG6VNhH5aJNy9sT7jUQkgYP5GeGPUvw?usp=sharing)
```
Comment (claudiofantacci) [inline examples/README.md:16]: Change `[google colab](https://colab.research.google.com/drive/1EJG6VNhH5aJNy9sT7jUQkgYP5GeGPUvw?usp=sharing)` to `[Colab](./image_augmentation.ipynb)`

**Comment by SupreethRao99** [comment]
Hey @claudiofantacci I've renamed the file to `image_augmentation.ipynb` and saved it under ` /examples` as requested. The link the notebook will remain the same.

Could you take a look and tell me if any other changes need to be made? Thank you !

**Comment by claudiofantacci** [comment]
Hey @SupreethRao99 thanks a lot again!
Please also update the link to point to the internal `ipynb`, which would make the repo self-contained 😄

File: examples/README.md (hunk 1 above)
Comment (SupreethRao99) [reply examples/README.md:16]: ill change it right away

File: examples/README.md [hunk 2]
```diff
@@ -13,3 +13,4 @@ Then, to run the examples, you can simply run the following commands:
 $ cd examples/
 $ python image_augmentation.py
 ```
+You can also view an interactive version of the examples on [Colab](./image_augmentation.ipynb)
```
Comment (claudiofantacci) [inline examples/README.md:16]: Nit: add a dot at the end of the sentence, i.e. `You can also view an interactive version of the examples on [Colab](./image_augmentation.ipynb).`

**Review by claudiofantacci — CHANGES_REQUESTED** [review]

**Review by claudiofantacci — APPROVED** [review]
LGTM!
Thanks for contributing!

**Review by claudiofantacci — CHANGES_REQUESTED** [review]
Hey @SupreethRao99 could you please open for edit `image_augmentation.ipynb` with any text editor and add a single empty line at the bottom of the file?

**Comment by SupreethRao99** [comment]
Hey, @claudiofantacci I've added a single empty line at the end of `image_augmentation.ipynb`.

**Review by claudiofantacci — APPROVED** [review]

**Comment by claudiofantacci** [comment]
The PR will be automatically merged by Copybara automatically. This may take a while, sorry about that 😄
If anything goes wrong, I'll let you know!
In the meantime @SupreethRao99 thanks again for the contribution 🚀

**Comment by SupreethRao99** [comment]
Thank you ! @claudiofantacci

**Pull request merged**